projects/addon-editor/components/editor/editor.module.ts

```diff
@@ -29,8 +34,10 @@ import {TuiEditorComponent} from './editor.component';
         TuiToolbarModule,
         TuiEditLinkModule,
         TuiDesignModeModule,
+        TuiEditorNewModule,
+        TuiToolbarNewModule,
     ],
     declarations: [TuiEditorComponent],
-    exports: [TuiEditorComponent],
+    exports: [TuiEditorComponent, TuiEditorNewComponent],
```

Comment:
We should probably do the same with `DataListWrapper` and `DataListDirective` so people will not have to import both modules.